src/Patient/Information/index.js

```diff
@@ -1,209 +1,69 @@
-            patientUIStore.router.push(steps[props.step].push);
-            patientUIStore.goToWalkThrough(props.step)
-
-        }}>
-        {props.children}
-    </Button>)
-}
-
-const Info = observer(() => {
-
-    const { t } = useTranslation('translation');
-    const classes = useStyles();
-    const { patientUIStore, patientStore } = useStores();
-    const surveyLink = window._env.REDCAP_EOT_SURVEY_LINK || "";
-    const pushEnabledState = usePushEnabled();
-    const surveyAvailable = patientStore.patientInformation.weeksInTreatment >= 20;
-
-    const history = useHistory();
-
-
-    return (
-        <div className={classes.container}>
-            {pushEnabledState != 'granted' && <Section highlight={Colors.highlightYellow} expanded={patientUIStore.onPushEnrollmentInstructions} title={<><HelpIcon />{t('notificationInstructions.steps.title')}</>}>
-                <NotificationInstructions />
-            </Section>}
-            <Section title={<><LiveHelpIcon />{t('patient.information.helpSection')}</>}>
-                <HelpSection />
-            </Section>
-            {surveyAvailable && <Section title={<><ListAltIcon />{t('archive.patientSide.surveyButton')}</>}>
-                <div className={classes.survey}>
-                    <Typography variant="body1">{t('archive.patientSide.survey')}</Typography>
-                    <NewButton href={surveyLink} icon={<ListAltIcon />} text={t('appSurvey.goToSurvey')} />
-                </div>
-            </Section>}
-            <Section title={<><VideoIcon />{t('patient.information.videos')}</>}>
-                <Videos />
-            </Section>
-            <Section title={<><InfoIcon />{t('patient.information.education')}</>}>
-                <TreatmentMessages />
-            </Section>
-            <Section onClick={()=>{history.push("/test-instructions")}} title={<><TestIcon />{t('patient.information.testInstructions')}</>}>
-
-            </Section>
-            <Section title={<><HelpIcon />{t('patient.information.questions')}</>}>
-                <Markdown options={{ overrides: { Drawer: { component: MarkdownRender } } }} children={file} />
-            </Section>
-            <Section title={<><ErrorIcon />{t('patient.information.techSupport')} / <br /> {t('patient.information.reportIssue')}</>}>
-                <ErrorReporting />
-            </Section>
-            <Section title={<><TrackChangesIcon />{t('patient.information.changeLog')}</>}>
-                <ChangeLog />
-            </Section>
-            <VersionNumber />
-        </div>
-    )
+const useStyles = makeStyles({
+    grid:{
+        display: "grid",
+        gridTemplateColumns: "1fr 1fr",
+        gridGap: "16px"
+    },
+    box:{
+        border: "solid 1px lightgray",
+        borderRadius: "4px",
+        display: "flex",
+        aspectRatio: "1",
+        alignItems: "center",
+        justifyContent: "center",
+        "& *":{
+            textAlign: "center"
+        }
+    }
```

Comment:
We loves the grid. Makes constraining sizes much easier as opposed to flex. I think this new settings page is a perfect place to implement the grid.  The only changes I would make would be responsive ones, which I know aren't very necessary on this project.